akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -272,11 +272,11 @@ final case class RootActorPath(address: Address, name: String = "/") extends Act
 
   override val toSerializationFormat: String = toString
 
-  override def toStringWithAddress(addr: Address): String =
+  override def toStringWithAddress(newAddress: Address): String =
```

Comment:
Ok, I got it. Thanks for you explanation! : ) I will rollback this soon. BTW, maybe it would be a good idea to send an issue to MiMa? They would know this situation.